templates/perl/pkgs/default.nix

```diff
@@ -0,0 +1,14 @@
+{
+  lib,
+  perlPackages,
+}:
+perlPackages.buildPerlPackage rec {
+  pname = "my-package";
+  version = "0.0.0";
+
+  src = ../.;
+
+  outputs = ["out"];
```

Comment:
I'm curious why does Perl need this?

Reply:
@mkenigs Perl actually may not. I can try it tomorrow without